CONTRIBUTING.md

```diff
@@ -163,11 +163,11 @@ start with, then the file name `index.md` should remain.
 1. Edit your `index.md` file based on the guidance in comments in that file -
 you'll find lots of helpful examples in other extension files. If you are
 documenting a Kong plugin, be sure to see the next section.
-1. If you have a custom logo, add a square-format PNG file to
+1. Custom logos are required for publication on the Kong plugin hub. If you don't have a custom logo yet, please duplicate
+an existing default logo file, and rename it as noted above. Custom logos should be a square-format PNG file, 
```

Comment:
```suggestion
1. Custom logos are required for publication on the Kong plugin hub. Custom logos 
should be a square-format PNG file, 
```
I think we should remove this reference to the default icon; it encourages them to not provide a logo if they see that there's an alternative.